lib/seek/samples/attribute_type_handlers/seek_sample_multi_attribute_type_handler.rb

```diff
@@ -0,0 +1,60 @@
+module Seek
+  module Samples
+    module AttributeTypeHandlers
+      class SeekSampleMultiAttributeTypeHandler < SeekResourceAttributeTypeHandler
+        class MissingLinkedSampleTypeException < AttributeHandlerException; end
+
+        def type
+          'Sample'
+        end
+
+        def test_value(value)
+          if value.kind_of?(Array)
+            value.each {|v| test_value_item v}
+          else
+            test_value_item value
+          end
+        end
+
+        def convert(value)
+          if value.kind_of?(Array)
+            value.uniq.map {|v| get_conversion v}
+          else
+            if value.include? ','
```

Comment:
You don't need this check, since if you call `split` on a string that does not contain the value, it just returns itself in an array.

Can also reduce duplication by first making sure `value` is an array, then calling `map` etc.